controlplane/src/core/test-util.ts

```diff
@@ -111,6 +112,9 @@ export async function seedTest(queryConnection: postgres.Sql, userTestData: User
 export function createTestContext(
   organizationName = 'wundergraph',
   organizationId = randomUUID(),
+  isAdmin = true,
+  hasWriteAccess = true,
+  roles: ('admin' | 'developer' | 'viewer')[] = ['admin'],
```

Comment:
We should create a type and reuse in both places.

Reply:
Done